CommonLibSF/include/RE/T/TESObjectCELL.h

```diff
@@ -12,7 +13,7 @@ namespace RE
 	public:
 		SF_RTTI_VTABLE(TESObjectCELL);
 		SF_FORMTYPE(CELL);
-
-		//
+		std::uint8_t        pad[0xD8];
```

Comment:
Minor nitpick, add an empty line after the `SF_FORMTYPE` macro so it's consistent with the rest of the RE'd headers

Reply:
missed it, thanks!